docs/DeveloperGuide.md

```diff
@@ -103,9 +103,10 @@ Here's a (partial) class diagram of the `Logic` component:
 
 How the `Logic` component works:
 1. When `Logic` is called upon to execute a command, it uses the `AddressBookParser` class to parse the user command.
-1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `AddCommand`) which is executed by the `LogicManager`.
-1. The command can communicate with the `Model` when it is executed (e.g. to add a person).
-1. The result of the command execution is encapsulated as a `CommandResult` object which is returned back from `Logic`.
+2. If the view or add command is executed, in addition to the `AddressBookParser` class, next `Logic` uses the `ContactDetailsParser` class to parse the user command. 
+3. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `AddCommand`) which is executed by the `LogicManager`. The only commands whose creation is specific to the `ContactDetailsParser` class are the `EditCommand` ,`DeleteFieldCommand` and `BackCommand`  classes. On the other hand, the `AddCommmand`, `ListCommand`, `ViewCommand`, `DeleteCommand`, `FindCommand`, `ClearCommand`, `MeetCommand`, `UpdateCommand`, `CancelCommand` and `CancelAllCommand` classes are specific to the `AddressBookParser` class. General commands applicable to both parsers are the `ExitCommand` and `HelpCommand` classes. 
```

Comment:
Point 3 is a little bit too wordy. Perhaps you can consider phrasing it some other way?